FrontEnd/src/components/profileList/ErrorPage.jsx

```diff
@@ -0,0 +1,38 @@
+import css from "./ErrorPage.module.css";
+import { Button, ConfigProvider } from "antd";
+
+export default function ErrorPage() {
+  return (
+    <div className={css.page}>
+      <div className={css.container}>
+        <div className={css.text404}>404</div>
+        <div className={css.block}>
+          <div className={css.explanation}>
+            <p className={css.main}>Щось пішло не так</p>
+            <p className={css.details}>
+              Схоже, це неправильна адреса, ця сторінка видалена, перейменована
+              або тимчасово недоступна.
+            </p>
+          </div>
+          <ConfigProvider
```

Comment:
No need to write it for every component `ConfigProvider` could be moved to the root of the application, so all buttons will have the same style

Reply:
It is done in another PR (#246 ). This page is temporary here. It will be removed as soon as PR #246 will be merged.